en/app.ftl

```diff
@@ -166,6 +168,36 @@ landing-use-cases-gaming-body = Enjoy online gaming, but worried about other gam
 landing-faq-headline = Top questions about { -brand-name-firefox-relay }
 landing-faq-cta = See more FAQs about { -brand-name-firefox-relay }
 
+## Premium promo page
+
+premium-promo-hero-headline = Make protecting your inbox easier with { -brand-name-firefox-relay-premium }
+# Variables:
+#   $monthly_price (string) - the monthly cost (including currency symbol) for Relay Premium. Examples: $0.99, 0,99 €
+premium-promo-hero-body-html = With { -brand-name-firefox-relay-premium }, you get unlimited custom email aliases that forward only the emails you want to your real email address. <b>For a limited time, you can upgrade to Relay Premium for only { $monthly_price } a month.</b>
+premium-promo-hero-cta = Upgrade now
+premium-promo-availability-warning = { -brand-name-relay-premium } is curently available in the USA, Germany, United Kingdom, Canada, Singapore, Malaysia, New Zealand, France, Belgium, Austria, Spain, Italy, Switzerland, Netherlands, Ireland.
```

Comment:
Should we add `and` before `Ireland`? 

Also, why do we list the countries in this order? I know once localized, it will not be in any kind of order in the target languages. I just find this list all over the place, :).

Reply:
Yeah I was thinking about that too, but then that would get us in Oxford comma territory and decided not to change it. I'll do a double-check with content though :)


I think the order is in order of when we expanded to each country - the Netherlands and Ireland were the latest additions.

Reply:
@Vinnl just noticed a typo in the word `currently`

Reply:
Alright, it's now in alphabetical order, with an "and" and an Oxford comma. Also, currently is now spelled with two `r`s, and the Netherlands and the UK now have "the" in front of them.

Reply:
Thanks for the fixes!! Good to go now! Officially approved!